code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -668,7 +668,7 @@ \subsubsection{Data Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           NFL=\frac{{\hat{q}_{\text{tol}}} E h^{4}}{\left(a b\right)^{2}}
+           NFL=\frac{{\hat{q}_{\text{tol}}} E h^{4.0}}{\left(a b\right)^{2}}
```

Comment:
whereas this one's odd.  fourth powers are rare - but it might be exact too.

Reply:
For the moment, I changed this one back to the exact 4